### ISSUE TITLE
How can I invoke the lambda function locally

### ISSUE THREAD

**Issue by casertap** [akrylysov/algnhsa#23]
Hi,
Before I try to use algnhsa, I was having implementing something similar to this:
https://djhworld.github.io/post/2018/01/27/running-go-aws-lambda-functions-locally/

to query my lambda function locally.

Is algnhsa still allow using the _LAMBDA_SERVER_PORT to allow querying the lambda function locally using `client, err := rpc.Dial("tcp", ":8008")`

Thanks

**Comment by nuqz** [comment]
> Is algnhsa still allow using the _LAMBDA_SERVER_PORT to allow querying the lambda function locally using client, err := rpc.Dial("tcp", ":8008")

Yes, it does. `algnhsa` uses the `lambda.StartHandler(...)` function internally. [Here](https://github.com/aws/aws-lambda-go/blob/fb4cb6fb90f758b220231bc74a0e0971f56af019/lambda/entry.go#L50) we can see, that it takes `_LAMBDA_SERVER_PORT` from environment.
